src/store/courses/reducer.js

```diff
@@ -0,0 +1,45 @@
+import { ADD_COURSE, DELETE_COURSE, SAVE_COURSES } from './actionTypes';
+
+const coursesInitialState = [];
+
+export default function coursesReducer(
+	state = coursesInitialState,
+	{ type, payload }
+) {
+	switch (type) {
+		case SAVE_COURSES: {
+			return payload;
```

Comment:
SAVE_COURSES is basically GET_COURSES

![image](https://user-images.githubusercontent.com/28926420/154776297-957364d2-063d-4298-976b-0a15dbdaf1ae.png)

Reply:
Got it!